libs/s3/s3_misc.py

```diff
@@ -406,5 +406,62 @@ def get_total_used(access_key: str, secret_key: str, **kwargs):
 
     size =0
     for bucket in s3_resource.buckets.all():
-        size += get_objects_size_bucket(bucket.name,access_key, secret_key, **kwargs)[1]
+        size += get_objects_size_bucket(bucket.name, access_key, secret_key, **kwargs)[1]
     return size
+
+
+def copy_object(access_key: str, secret_key: str, src_bkt: str = None, src_obj: str = None,
+                dest_bkt: str = None, dest_obj: str = None, **kwargs):
+    """
+    Copy of an object that is already stored in Seagate S3 with different permissions.
+    """
```

Comment:
Add args & keywords in docstring
```suggestion
    """
    Copy src_bkt from src_objto to dest_bkt as dest_obj with different permissions.
    
    :param access_key: Access Key
    :param secret_key: Secret Key
    :param src_bkt: Source Bucket
    :param src_obj: Source Object
    :param dest_bkt: Destination Bucket
    :param dest_obj: Destination Object
    :keyword endpoint_url: Endpoint URL, default value S3_CFG["s3_url"]
    """
```